README.md

```diff
@@ -228,13 +237,64 @@ begin
 end Main;
 ```
 
-## Dependencies
+### Using the Python Library
+
+The following code shows how PyRFLX can be used to parse and generate messages in Python:
+
+
+```Python
+import sys
+
+from rflx.pyrflx import MessageValue, PyRFLX
+
+PYRFLX = PyRFLX(["specs/tlv.rflx"])
+TLV = PYRFLX["TLV"]
+
+
+def parse_message(input_bytes: bytes) -> MessageValue:
+    msg = TLV["Message"]
+    msg.parse(input_bytes)
+    return msg
+
+
+def create_message() -> MessageValue:
+    msg = TLV["Message"]
+    msg.set("Tag", "Msg_Data")
+    msg.set("Length", 4)
+    msg.set("Value", b"\x01\x02\x03\x04")
+    return msg
+
 
-- [Python >=3.6](https://www.python.org)
+if parse_message(b"\x40\x04\x01\x02\x03\x04") != create_message():
+    sys.exit("Error")
+```
+
+## Installation
+
+RecordFlux can be installed from PyPI:
+
+```Console
+pip install RecordFlux
```

Comment:
How do we want to handle the `pip` vs. `pip3` issue? On my system the above command gives:

```
$ pip install RecordFlux
Collecting RecordFlux  Could not find a version that satisfies the requirement RecordFlux (from versions: )
No matching distribution found for RecordFlux
```

Maybe we want to use `pip3` to be on the safe side? AFAICT this should also work on Arch, but I'm not sure regarding other distros...

Reply:
Fixed in be3824d.